frontend/public/reducers/monitoring.ts

```diff
@@ -52,7 +52,7 @@ export const alertSeverityOrder = (alert: Alert): ListOrder => {
 
 // Sort alerts and silences by their state (sort first by the state itself, then by the timestamp
 // relevant to the state)
-export const alertStateOrder = (alert: Alert): ListOrder => [
+export const alertStateOrder = (alert): ListOrder => [
```

Comment:
Looks like `: Alert` was accidentally removed

Reply:
Ah, yes. Added it back.